contentful/apps
# Feature/integ 622: Add AWS Build button to Sidebar Location

## Purpose

Content editors will need a way to trigger AWS Amplify builds from the sidebar. This PR adds the Build button to the sidebar location. 

## Approach
For testing purposes and to get around the CORS errors thrown from Amplify, I used [cors-anywhere](https://www.npmjs.com/package/cors-anywhere) to proxy the request from localhost to a CORS enabled server. This is meant only for development purposes. 

## Testing steps

- Create an AWS amplify app and point it to your localhost
- Go to any Content screen, verify Build button shows up in the sidebar
- Execute Build button, ensure appropriate error messaging occurs
- To test success messaging, follow instructions in the Readme to enable `cors-anywhere`
